dbfit-java/build.gradle

```diff
@@ -83,7 +83,75 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task deploymentProperties {
+        doLast {
+            ext {
+                delimiterType = 'normal'
+                delimiter = ';'
+                setUpTestScriptPath = "${rootProject.projectDir}/test_vm/scripts/${project.name}/sql/"
+                setUpTestDbScript = setUpTestScriptPath + "create-databases-${project.name}.sql"
+                tearDownTestDbScript = setUpTestScriptPath + "drop-databases-${project.name}.sql"
+            }
+        }
+    }
+
+    // to be overriden when needed
+    ext.connectionPropertiesFiles = []
+
+    task connectionFileProperties {
+        doLast {
+            copyConnectionProperties '.'
+            ext {
+                userName = ''
+                passWord = ''
+                service = ''
+                database = ''
```

Comment:
Do we need to define these defaults explicitly? In any case - if needed, we already have assignments below: maybe they can be reused (out of empty properties map)